liTest run started
liTest org.scalajs.junit.ExceptionAfterAssume.assumeFail started
leTest org.scalajs.junit.ExceptionAfterAssume.assumeFail failed: This assume should not pass, took <TIME>
e2org.scalajs.junit.ExceptionAfterAssume.assumeFail
leTest org.scalajs.junit.ExceptionAfterAssume.assumeFail failed: after() must be called, took <TIME>
ldTest org.scalajs.junit.ExceptionAfterAssume.assumeFail finished, took <TIME>
liTest run finished: 2 failed, 0 ignored, 1 total, <TIME>
d
